zeroex/orderwatch/order_watcher_test.go

```diff
@@ -688,3 +635,18 @@ func setupSubTest(t *testing.T) func(t *testing.T) {
 		blockchainLifecycle.Revert(t)
 	}
 }
+
+func waitForOrderEvents(t *testing.T, orderEventsChan <-chan []*zeroex.OrderEvent, timeout time.Duration) []*zeroex.OrderEvent {
+	start := time.Now()
+	select {
+	case orderEvents := <-orderEventsChan:
+		fmt.Println("orderEvents!", orderEvents)
+		return orderEvents
+	case <-time.After(timeout):
+		elapsed := time.Since(start)
+		fmt.Println("elapsed", elapsed)
+		fmt.Println("timeout!")
+		t.Fatal("timed out waiting for order events")
+	}
+	return []*zeroex.OrderEvent{}
+} 
```

Comment:
nit: There's no new line at the end of this file. Do you have your editor configured to automatically add them?

Reply:
I have `    "files.insertFinalNewline": true,`